deps.edn

```diff
@@ -5,35 +5,44 @@
         org.scicloj/kindly {:mvn/version "4-beta12"}
         generateme/fastmath {:mvn/version "3.0.0-alpha1"}
         aerial.hanami/aerial.hanami {:mvn/version "0.20.0"}
-        org.scicloj/hanamicloth {:mvn/version "1-alpha8"
-                                 :exclusions [scicloj/metamorph.ml]}
-        scicloj/metamorph.ml {:git/url "https://github.com/scicloj/metamorph.ml.git"
-                              :git/sha "50f47dad934a2786b3cf025bef509f1f3d1a7e1d"
-                              :exclusions [generateme/fastmath]}
-        org.scicloj/scicloj.ml.tribuo {:mvn/version "0.1.1-branch-noj-2-alpha4-SNAPSHOT"
-                                       :exclusions [scicloj/metamorph.ml]}
-        org.tribuo/tribuo-regression-sgd {:mvn/version "4.3.1"}
-        org.tribuo/tribuo-regression-tree {:mvn/version "4.3.1"}
-        org.tribuo/tribuo-regression-xgboost {:mvn/version "4.3.1"}
-        org.tribuo/tribuo-classification-sgd {:mvn/version "4.3.1"}
-        org.tribuo/tribuo-classification-tree {:mvn/version "4.3.1"}
-        org.tribuo/tribuo-classification-xgboost {:mvn/version "4.3.1"}
+        org.scicloj/hanamicloth {:mvn/version "1-alpha8"}
+        org.scicloj/metamorph.ml {:mvn/version "0.9.0"}
+        org.scicloj/scicloj.ml.tribuo {:mvn/version "0.1.2"}
```

Comment:
v 0.1.2 still uses tribuo 4.2.0
so I removed as well your additional 4.3.1 dependencies.

I propose we merge, then I do a 0.1.3 release of tribuo depending on 4.3.1 and then we re-add the dependencies.
I can them move them from "test" deps into real deps

Reply:
You are right, thanks.